src/components/Schedule.js

```diff
@@ -39,15 +39,20 @@ const Schedule = () => {
   const [isOpen, setIsOpen] = useState(false);
   const [clickedIdx, setClickedIdx] = useState(-1);
   const [width, setWidth] = useState(window.innerWidth);
+  const [currentTime, setCurrentTime] = useState(new Date());
 
   useEffect(() => {
     const updateSize = () => {
       setWidth(window.innerWidth);
     };
+    const interval = setInterval(() => setCurrentTime(new Date()), 1000);
     window.addEventListener("resize", updateSize);
     updateSize();
-    return () => window.removeEventListener("resize", updateSize);
-  }, []);
+    return () => {
+      window.removeEventListener("resize", updateSize);
+      clearInterval(interval);
+    };
+  }, [currentTime]);
```

Comment:
Could you move the interval stuff to a new useEffect, so that it doesn't also reset eventlistener every second?